bin/pygrb/pycbc_pygrb_plot_injs_results

```diff
@@ -105,6 +106,21 @@ def complete_mass_data(injs, key, tag):
     return data
 
 
+def complete_spin_data(injs, key, tag):  
+    """Extract data related to effective precession spin from raw injection data"""
+
+    if key in ('chi_p', 'chip'):
+        mass1 = injs[tag+'/mass1']
+        mass2 = injs[tag+'/mass2'] 
+        spin1x = injs[tag+'/spin1x']
+        spin1y = injs[tag+'/spin1y']
+        spin2x = injs[tag+'/spin2x']
+        spin2y = injs[tag+'/spin2y']
+
+        data = pycbc.conversions.chi_p(mass1, mass2, spin1x, spin1y, spin2x, spin2y)
+        return data
```

Comment:
We only have one possible case to handle at the moment, so you can remove the if statement.